src/Bicep.Core/FileSystem/InMemoryFileResolver.cs

```diff
@@ -85,6 +85,12 @@ public bool TryDirExists(Uri fileUri)
                 .Any(key => key.ToString().StartsWith(fileUri.ToString()));
         }
 
+        public bool TryFileExists(Uri fileUri)
```

Comment:
>TryFileExists [](http://example.com/codeflow?start=20&length=13)

Why not
```csharp
return this.fileLookup.ContainsKey(uri);
```

(the scan in `TryDirExists` was needed to do a prefix match, but I think we can avoid it here)

Reply:
I ended up adding `FileExists` in my own PR btw, so now they're duplicated.